agent/proto/agentpb/agent.proto

```diff
@@ -172,4 +181,7 @@ service Agent {
   rpc Time(TimeRequest) returns (TimeResponse) {}
   // Timeline collects the status timeline
   rpc Timeline(TimelineRequest) returns (TimelineResponse) {}
+  // UpdateTimeline requests a new event be added to the timeline.
+  // This RPC is idempotent. Repeated calls will have no effect.
```

Comment:
Repeated as in with the same event value?

Reply:
Yes, will specify.